webapp/security/views.py

```diff
@@ -507,31 +507,49 @@ def cve_index():
     versions = flask.request.args.getlist("version")
     statuses = flask.request.args.getlist("status")
 
+    security_api = SecurityAPI(
+        session=session,
+        base_url="http://192.168.64.6:8030/security/",
+    )
+
+    cves_query = security_api.get_cves()
+
     is_cve_id = re.match(r"^CVE-\d{4}-\d{4,7}$", query.upper())
 
-    if is_cve_id and db_session.query(CVE).get(query.upper()):
+    if is_cve_id and cves_query.get(query.upper()):
         return flask.redirect(f"/security/{query.lower()}")
 
-    all_releases = (
-        db_session.query(Release)
-        .order_by(desc(Release.release_date))
-        .filter(Release.codename != "upstream")
-        .all()
-    )
+    # releases in desc order
+    releases_json = security_api.get_releases().get("releases")
```

Comment:
I'd suggest that you move the `.get("releases")` part into the `security_api.get_releases()` function, so that `get_releases` literally returns a list of releases. Makes things less surprising.

``` python3
def get_releases():
    # ...
    return response.json().get("releases")
```